src/Rules/Extensions/ExtensionMethods.cs

```diff
@@ -50,7 +51,7 @@ public static Condition WithTimer(this Condition c, string message)
                 var stopwatch = Stopwatch.StartNew();
                 bool retVal = c.Matches(e);
                 var ms = stopwatch.ElapsedTicks / (float)10000;
-                Debug.WriteLine(Invariant($"{ms} ms {message}"));
+                Debug.WriteLine(string.Format(CultureInfo.CurrentCulture, ConditionDescriptions.MillisecondsFormat, ms, message));
```

Comment:
This was originally added for debugging. We should probably just remove the line